docs/getting-started.md

```diff
@@ -211,14 +212,21 @@ bazel run -c dbg --run_under="valgrind --tool=callgrind --callgrind-out-file=/tm
 
 ## Flashing And Debugging A STM32F4 MCU
-1. Make sure you've followed [Installing Firmware Dependencies](#installing-firmware-dependencies), and have a STM32F4 based main board or radio dongle plugged into your computer. Do not plug both the dongle and the robot at the same time!
+1. Make sure you've followed [Installing Firmware Dependencies](#installing-firmware-dependencies), and have a STM32F4 based main board plugged into your computer. Do not plug both the dongle and the robot at the same time!
 2. Make sure the robot is elevated, with the wheels not touching any surface to avoid experimental firmware causing accidental damage.
-3. From the `src` folder, to flash the radio dongle, run `bazel run --cpu=stm32f4 //firmware/tools:legacy_flash_firmware radio_dongle`. Replace `radio_dongle`, with `robot` to flash the robot.
-4. For the robot, make sure the robot is in bootloader mode (BL switch on the piano keys is down), and push the power switch away from the dribbler (i.e towards the back of the robot) and hold it there before running the command. The dongle can be simply be plugged in with no additional action.
+3. From the `src` folder, to flash the radio dongle, run `bazel run --cpu=stm32f4 //firmware/tools:legacy_flash_firmware robot`
+4. For the robot, make sure the robot is in bootloader mode (BL switch on the piano keys is down), and push the power switch away from the dribbler (i.e towards the back of the robot) and hold it there before running the command.
 5. There should be a progress bar indicating the flashing progress, hold the switch back until the process finishes.
 6. When the process is finished, release the power switch, push the BL switch back up, and the robot now has been flashed!
 7. To see print outs from the robot, run `sudo cat /dev/ttyACM0`. If `ttyACM0` isn't the right device, run `ls /dev/tty*` with the robot disconnected, and again with the robot connected, and replace `ttyACM0` with the new device that has been added. The radio dongle does not have this feature.
 
+## Flashing the Radio Dongle
+1. Make sure you've followed [Installing Firmware Dependencies](#installing-firmware-dependencies), and have a dongle plugged into your computer. Do not plug both the dongle and the robot at the same time!
+2. From the `src` folder, to flash the radio dongle, run `bazel run --cpu=stm32f4 //firmware/tools:legacy_flash_firmware radio_dongle`.
+3. If the dongle has a blue pcb, then follow step (a). If the dongle has a green pcb then follow step (b). There should be a progress bar indicating the flashing progress.
+    (a) Hold down the red button while unplugging and plugging in the dongle. There should be a single orange light on. Then run the script described in step 2.
+    (b) Hold down the yellow button while running the script described in step 2.
```

Comment:
![image](https://user-images.githubusercontent.com/22976042/101132386-caa68800-35bb-11eb-89ec-8d255c77019c.png)

This didn't render well on my screen

Reply:
good catch fixed